docs/src/heaps.md

```diff
@@ -18,6 +18,9 @@ push!(h, v)       # add a value to the heap
 top(h)            # return the top value of a heap
 
 pop!(h)           # removes the top value, and returns it
+
+delete!(h,id)     # removes the node having node_id as id from heap 
+
```

Comment:
This should be moved to the mutable heap description below, since it's not a method of all heaps.  It should also be added to comment at the top of `heaps.jl`.